src/lib/store_email_signature.ts

```diff
@@ -26,6 +27,13 @@ export async function processAndStoreEmailSignature(
 		console.timeEnd('verificationResult')
 		console.timeEnd('processAndStoreEmailSignature')
 		console.log(chalk.redBright('Error verifying DKIM signature:\n Domain: ', tags.d, '\n', error));
+		if (
+			error instanceof Error &&
+			error.message &&
+			error.message.includes("bad signature")
```

Comment:
Where is this string, "bad signature", generated? I can't find it anywhere else than here.

Reply:
I see, thanks!